pubsub/tests/unit/pubsub_v1/subscriber/test_policy_thread.py

```diff
@@ -115,6 +116,7 @@ def test_on_response():
     # Actually run the method and prove that the callback was
     # called in the expected way.
     policy.on_response(response)
+    time.sleep(1)
```

Comment:
Rather than adding a potentially flaky `time.sleep`, ISTM that a better test would be to supply a mock for the `executor` argument to the policy, and then assert that the (mock) futures returned by its `submit` method have their `add_done_callback` called with the `_callback_completed`.

Reply:
Yep, agreed.  Will do.